FreeTubeApp/FreeTube
# Shorts tab on channel page

# Add 'Shorts' tab to the Channel page






## Pull Request Type


## Description
Adds the Shorts tab for a channel to the Channel Page. Allows for playing via direct or invidious api

## Screenshots

### Before:
![before](https://user-images.githubusercontent.com/361120/217639040-77d1c1f3-a6e3-419e-8b15-f8e3b2feccbc.png)

#### After:

![after](https://user-images.githubusercontent.com/361120/217639103-f9241a32-bf65-403f-819b-fa5bf602f72b.png)


## Testing

1) Find a channel which has Shorts and Subscribe to it.
2) View the Channels page, then click on that channel
3) The new "Shorts" tab should be visible. Click it
4) There should be a grid with Shorts in it. 
5) Click any of the shorts. You should be taken to the Watch page and be able to watch the Short
